### DISCUSSION TITLE
nvflare dashboard --start

### DISCUSSION THREAD

**Discussion by rekkles2** [NVIDIA/NVFlare#2082] [Q&A] [answered]
I'm using an AGX Xavier device, I already have the Docker environment and Jetpack installed on the device, and I want to use the NVFLARE Dashboard for a real-world project, Now I have successfully used nvflare dashboard --start and successfully pulled the dashboard image, but I don't know how to open the web of the dashboard ui to distribute the package. I am a beginner and request your help and guidance.There are several possibilities that I can guess. The first one is that the AGX Docker comes with nvidai Docker. The second one is that the dashboard UI must be started through the Docker desktop app

**Comment by YuanTingHsieh** [comment, marked as answer]
If your AGX Xavier device has a browser, you can just open the browser and check http://0.0.0.0:8443 for the dashboard UI.
If your AGX Xavier device does not have a browser in it, you find its ip first, then you can use other machine that has a browser and can connect to AGX Xavier device via internet, and then you open the browser and check http://[Xavier device ip]:8443 for the dashboard UI.

@IsaacYangSLA can add more details or correct me, thanks

**Reply by rekkles2** [reply]
Thank you very much for your help
